cardano-testnet/src/Testnet/Start/Types.hs

```diff
@@ -82,14 +85,27 @@ newtype NodeConfigurationYaml = NodeConfigurationYaml
   { unYamlFilePath :: FilePath
   } deriving (Eq, Show)
 
-newtype Conf = Conf
+data Conf = Conf
   { tempAbsPath :: TmpAbsolutePath
-  } deriving (Eq, Show)
+  , -- | Function to rewrite the Conway genesis file before creating the testnet
+    rewriteConway :: Ledger.ConwayGenesis StandardCrypto -> Ledger.ConwayGenesis StandardCrypto
+  }
+
+instance Show Conf where
+  -- We ignore rewriteConway
+  show Conf { tempAbsPath } =
+    unlines
+      ["Conf{"
+       , "  tempAbsPath=" ++ show tempAbsPath
+       , "  rewriteConway=unshowable"
+       , "}"
+      ]
 
 mkConf :: FilePath -> Integration Conf
 mkConf tempAbsPath' =
   return $ Conf
     { tempAbsPath = TmpAbsolutePath tempAbsPath'
+    , rewriteConway = id -- By default, don't change anything
```

Comment:
Can you link to the PR where you use it?

Reply:
No PR yet, but here's the WIP branch using this: https://github.com/IntersectMBO/cardano-node/blob/smelc/dvtUpdateToConstitutionTest/cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/ProposeNewConstitutionDRep.hs#L86